src/nvim/README.md

```diff
@@ -118,73 +119,77 @@ Example using bpftrace to track slow vim functions, and print out any files
 that were opened during the trace. At the end, it prints a histogram of
 function timing:
 
-    #!/usr/bin/env bpftrace
-
-    BEGIN {
-      @depth = -1;
-    }
-
-    tracepoint:sched:sched_process_fork /@pidmap[args->parent_pid]/ {
-      @pidmap[args->child_pid] = 1;
-    }
-
-    tracepoint:sched:sched_process_exit /@pidmap[args->pid]/ {
-      delete(@pidmap[args->pid]);
-    }
-
-    usdt:build/bin/nvim:neovim:eval__call_func__entry {
-        @pidmap[pid] = 1;
-        @depth++;
-        @funcentry[@depth] = nsecs;
-    }
-
-    usdt:build/bin/nvim:neovim:eval__call_func__return {
-        $func = str(arg0);
-        $msecs = (nsecs - @funcentry[@depth]) / 1000000;
-
-        @time_histo = hist($msecs);
-
-        if ($msecs >= 1000) {
-          printf("%u ms for %s\n", $msecs, $func);
-          print(@files);
-        }
-
-        clear(@files);
-        delete(@funcentry[@depth]);
-        @depth--;
-    }
-
-    tracepoint:syscalls:sys_enter_open,
-    tracepoint:syscalls:sys_enter_openat {
-      if (@pidmap[pid] == 1 && @depth >= 0) {
-        @files[str(args->filename)] = count();
-      }
-    }
-
-    END {
-      clear(@depth);
-    }
-
-    $ sudo bpftrace funcslower.bt
-    1527 ms for Slower
-    @files[/usr/lib/libstdc++.so.6]: 2
-    @files[/etc/fish/config.fish]: 2
-    <snip>
-
-    ^C
-    @time_histo:
-    [0]                71430 |@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@|
-    [1]                  346 |                                                    |
-    [2, 4)               208 |                                                    |
-    [4, 8)                91 |                                                    |
-    [8, 16)               22 |                                                    |
-    [16, 32)              85 |                                                    |
-    [32, 64)               7 |                                                    |
-    [64, 128)              0 |                                                    |
-    [128, 256)             0 |                                                    |
-    [256, 512)             6 |                                                    |
-    [512, 1K)              1 |                                                    |
-    [1K, 2K)               5 |                                                    |
+```c
```

Comment:
?

Reply:
there is no dedicated bpf hightlighting identifier, I tried a few and c seems to do the best job at somewhat hightlighting this imo. But if you have a better idea, or think this should just stay without any language identifier I can remove it again.